From d50829cb7fadae3b07488d271e837e6e4179f582 Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Tue, 2 Jun 2020 16:56:16 +0800
Subject: [PATCH] arm64: dts: rockchip: add xhci-warm-reset-on-suspend-quirk
 for rk3399

This patch adds property "xhci-warm-reset-on-suspend-quirk"
for rk3399 DWC3 xHC to do a warm port reset for USB3 port
upon PM suspend.

This patch also deletes the unused property:
snps,usb3-warm-reset-on-resume-quirk

Change-Id: I0d50e1c603a3114f6bc5f822334546c6e1668b0e
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index 1b97c7d12da3..9b6aad894818 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -466,7 +466,7 @@
 			snps,dis-tx-ipgap-linecheck-quirk;
 			snps,xhci-slow-suspend-quirk;
 			snps,xhci-trb-ent-quirk;
-			snps,usb3-warm-reset-on-resume-quirk;
+			snps,xhci-warm-reset-on-suspend-quirk;
 			status = "disabled";
 		};
 	};
@@ -500,7 +500,7 @@
 			snps,dis-tx-ipgap-linecheck-quirk;
 			snps,xhci-slow-suspend-quirk;
 			snps,xhci-trb-ent-quirk;
-			snps,usb3-warm-reset-on-resume-quirk;
+			snps,xhci-warm-reset-on-suspend-quirk;
 			status = "disabled";
 		};
 	};
-- 
2.35.3

